test/config/utility.cc

```diff
@@ -148,6 +148,8 @@ std::string ConfigHelper::tlsInspectorFilter() {
   return R"EOF(
 name: "envoy.filters.listener.tls_inspector"
 typed_config:
+  "@type": type.googleapis.com/envoy.extensions.filters.listener.tls_inspector.v3.TlsInspector
+  enable_ja3_fingerprinting: true
```

Comment:
I think it would be preferable to revert this, and only enable this for tests that exercise this feature. Otherwise there is no longer any coverage for cases where this is false.